doc/enterprise/changelog.rst

```diff
@@ -0,0 +1,454 @@
+..  _enterprise_changelog:
+
+Changelog
+=========
+
+Versioning policy
+-----------------
+
+A Tarantool Enterprise Edition's SDK version consists of two parts:
```

Comment:
Not sure about this wording - `Tarantool Enterprise Edition's SDK`. Sounds a bit unnatural to me.
The old `Tarantool Enterprise SDK` sound better – it's not the same product (Tarantool EE), but a toolkit for creating application to work with it. Better to check with PMs.

Reply:
agree